controllers/grid/users/author/form/PKPAuthorForm.inc.php

```diff
@@ -26,13 +26,19 @@ class PKPAuthorForm extends Form {
 	 * Constructor.
 	 */
 	function __construct($publication, $author) {
-		parent::__construct('controllers/grid/users/author/form/authorForm.tpl');
+		$submission = Services::get('submission')->get($publication->getData('submissionId'));
+		$context = Services::get('context')->get($submission->getData('contextId'));
+
+		parent::__construct(
+			'controllers/grid/users/author/form/authorForm.tpl', 
+			true, 
+			$publication->getData('locale'), 
+			$context->getSupportedSubmissionLocaleNames()
```

Comment:
Are you sure this is the right method? I think you might want `$context->getSupportedSubmissionLocales()` but I'm not sure.

Reply:
Here I am the same function as in https://github.com/pkp/pkp-lib/blob/stable-3_3_0/classes/submission/form/SubmissionSubmitForm.inc.php#L44. 

By using the proposed by the PR function, the result is the following
![image](https://user-images.githubusercontent.com/560456/177132782-fedabdcc-d6f3-4874-b8a2-192a0f970f04.png)

By using the `$context->getSupportedSubmissionLocales()` the result is the following:
![image](https://user-images.githubusercontent.com/560456/177133007-03a1f136-47f4-4b26-b7d3-63cfe2cdae29.png)

The first (and proposed by the PR) seems the correct to me.

Reply:
:+1: You're right, thanks for looking into it.